app/controllers/users_controller.rb

```diff
@@ -0,0 +1,6 @@
+class UsersController < ApplicationController
+  def dashboard
+    @user_tasks = current_user.assigned_tasks.where(done: false).group_by(&:project)
```

Comment:
will be better to create a scope in a model (query should not leak outside model), it;s a bad practice. Define a scope and use it here